### PR TITLE
013 /docs/concepts/services networking/ingress.md

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -1,113 +1,145 @@
...
 -->
+## Ingress 是什么？
 
-Ingress 是允许连接到集群 Service 的规则集合。
+<!-- 
+Ingress exposes HTTP and HTTPS routes from outside the cluster to
+{{< link text="services" url="/docs/concepts/services-networking/service/" >}} within the cluster.
+Traffic routing is controlled by rules defined on the Ingress resource. 
+-->
+Ingress 公开了从集群外部到集群内部 {{< link text="services" url="/docs/concepts/services-networking/service/" >}} 的 HTTP 和 HTTPS 路由。
```
Comment (sftim) [inline content/zh/docs/concepts/services-networking/ingress.md]:
```suggestion
Ingress 公开了从集群外部到集群内部 {{< link text="Service" url="/docs/concepts/services-networking/service/" >}} 的 HTTP 和 HTTPS 路由。
```

File: content/zh/docs/concepts/services-networking/ingress.md [hunk 2]
```diff
@@ -611,64 +652,61 @@ Events:
...
-*更多 Ingress 控制器
-
-请跟踪 [L7 and Ingress proposal](https://github.com/kubernetes/kubernetes/pull/12827)以了解关于资源演化的更多细节，以及 [Ingress repository](https://github.com/kubernetes/ingress/tree/master) 以了解关于各种 Ingress 控制器演进的更多细节。
+<!--
+Track [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network)
+for more details on the evolution of Ingress and related resources. You may also track the
+[Ingress repository](https://github.com/kubernetes/ingress/tree/master) for more details on the
+evolution of various Ingress controllers.
+-->
+跟踪 [SIG 网络](https://github.com/kubernetes/community/tree/master/sig-network)以获得有关 Ingress 和相关资源演变的更多细节。您还可以跟踪 [Ingress 仓库](https://github.com/kubernetes/ingress/tree/master)以获取有关各种 Ingress 控制器的更多细节。
```
Comment (sftim) [inline content/zh/docs/concepts/services-networking/ingress.md]:
```suggestion
跟踪 [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network)以获得有关 Ingress 和相关资源演变的更多细节。您还可以跟踪 [Ingress 仓库](https://github.com/kubernetes/ingress/tree/master)以获取有关各种 Ingress 控制器的更多细节。
```
The name of that SIG is “SIG Network”.

File: content/zh/docs/concepts/services-networking/ingress.md [hunk 3]
```diff
@@ -611,64 +652,61 @@ Events:
 ```
 
 <!--
-You can achieve the same by invoking `kubectl replace -f` on a modified Ingress yaml file.
+You can achieve the same outcome by invoking `kubectl replace -f` on a modified Ingress YAML file.
 -->
-
 您可以通过 `kubectl replace -f` 命令调用修改后的 Ingress yaml 文件来获得同样的结果。
```
Comment (sftim) [inline content/zh/docs/concepts/services-networking/ingress.md]:
```suggestion
 您可以通过 `kubectl replace -f` 命令调用修改后的 Ingress YAML 文件来获得同样的结果。
```

File: content/zh/docs/concepts/services-networking/ingress.md (hunk 1 above)
Comment (tanjunchen) [reply content/zh/docs/concepts/services-networking/ingress.md]: Thanks

**Pull request merged**